src/desktop/apps/artwork/components/commercial/view.coffee

```diff
@@ -55,7 +55,7 @@ module.exports = class ArtworkCommercialView extends Backbone.View
         order = data?.createOrderWithArtwork?.orderOrError?.order
         location.assign("/order2/#{order.id}/shipping")
 
-    else
+    else if artwork.partner.type == "Auction" or artwork.partner.type == "Auction House"
```

Comment:
ooo @zephraph totally missed this, but in this context we'll have to access this like `@artwork.get('partner').type`

Reply:
(also ☕️ so `@artwork.partner?.type is 'Auction'` maybes)